src/components/02-molecules/Header/index.js

```diff
@@ -1,13 +1,16 @@
 import React, { PropTypes } from 'react';
+import { browserHistory } from 'react-router';
 import styles from './styles.scss';
 
+
 const Header = ({ user, logout }) => (
   <header className={styles.header}>
     <span className={styles.hello}>Hello</span>
     <span className={styles.name}>
       { user.name }!
     </span>
-    <span className={styles.logout} onClick={logout}>Log Out</span>
+    <span className={styles.link} onClick={() => browserHistory.push('/admin')}>Admin</span>
```

Comment:
Shouldn't this be using a Link component, rather than pushing to browserHistory directly?